libs/testing/src/user.session.ts

```diff
@@ -44,12 +36,13 @@ const EMAIL_BLOCK: IEmailBlock[] = [
 ];
 
 export class UserSession {
-  private environmentRepository = new EnvironmentRepository();
```

Comment:
We have the services with the dedicated functionalities to prepare the session so makes sense to use them rather than the DAL directly.